python/pysmurf/client/tune/smurf_tune.py

```diff
@@ -801,7 +801,7 @@ def find_peak(self, freq, resp, rolling_med=True, window=5000,
             band=None, subband=None, make_subband_plot=False,
             subband_plot_with_slow=False, timestamp=None, pad=50, min_gap=100,
             plot_title=None, grad_kernel_width=8, highlight_phase_slip=True,
-            amp_ylim=None):
+            amp_ylim=None, flip_phase=False, plot_phase=False):
```

Comment:
Missing doc string for the new `plot_phase` argument. @eyyoung24 Can you add it?

Reply:
Added docstring